playbooks/agile/team-working-agreement.md

```diff
@@ -0,0 +1,97 @@
+# Team working agreement
+
+The purpose of the working agreement is to ensure the team shares responsibility in defining expectations for how they will function together and enhance their self-organization process. It creates an awareness of both positive and negative behaviors that can impact the team.
+
+## Creating a team working agreement
+
+### Overview
+
+* This process uncovers the existing norms, both positive and negative, that impact team functioning.
+* The goal of a team working agreement is to reinforce positive behaviors while helping our team overcome dysfunctional ones.
+* A great set of questions to ask is:
+  * What expectations do team members have of each other?
+  * What is working well within our team?
+  * What is not working well?
+  * What should our team keep doing?
+  * What should our team start doing?
+  * What should our team stop doing?
+* Behaviors outlined in social contracts may include any positive behaviors that the team wants to encourage such as:
```

Comment:
The examples you've provided above are realistic items that may appear in each product team's working agreement, but I am hesitant to provide specific examples in this document because I don't want to frame the discussion for those teams. The working agreement should document how the members of a team agree to work together in harmony, and I don't think that the product of a team should be the main focus (although it probably will be included if necessary).